src/types/Sanity.d.ts

```
@@ -338,4 +332,4 @@ declare global {
     }
   }
 }
-export {};
\ No newline at end of file
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)